index.js

```diff
@@ -341,12 +341,25 @@ class GithubScm extends Scm {
                 }
             })
         );
-        const authorLookup = commitLookup.then(commitData =>
-            this.decorateAuthor({
+        const authorLookup = commitLookup.then((commitData) => {
+            const username = (commitData.author) ? commitData.author.login : '_none';
+
+            return this.decorateAuthor({
```

Comment:
If `commit.author` doesn't exist then why do we even bother with calling `this.decorateAuthor`? We can just return line 355.